plugins/Eq/EqEffect.cpp

```diff
@@ -131,13 +131,6 @@ bool EqEffect::processAudioBuffer( SampleFrame* buf, const fpp_t frames )
 	m_lp481.setParameters( sampleRate, lpFreq, lpRes, 1 );
 
 
-
```

Comment:
I had doubts with the above 14 lines, verifying that it is OK that these are now not executed if the Effect is not run, but prior to the PR they were. So to understand it, I made those 14 class members local and LMMS still compiled. Do you mind adding that change to the PR, so it gets more clear?

Reply:
It looks like they are member variables in order to cache the results of previous calculations and only recalculate if the parameters change in order to improve performance, so I think they should stay member variables.